### PR TITLE
Developer disc image for iOS 16.6 added

### PR DESCRIPTION
**Pull request by eyJhb**
Added new developer images.

At some point, we should do this automatically.. But unsure the best way to do it with that fuzzy search function.

### CODE REVIEW

File: ios/imagemounter/imagedownloader.go [hunk 1]
```diff
@@ -81,6 +81,7 @@ var (
...
 		"16.4.1":          "https://github.com/mspvirajpatel/Xcode_Developer_Disk_Images/blob/master/Developer%20Disk%20Image/16.4.1",
 		"16.5":            "https://github.com/mspvirajpatel/Xcode_Developer_Disk_Images/blob/master/Developer%20Disk%20Image/16.5",
+		"16.6":            "https://github.com/mspvirajpatel/Xcode_Developer_Disk_Images/blob/master/Developer%20Disk%20Image/16.6",
 	}
 
 	availableVersions = []string{"4.2", "4.3", "5.0", "5.1", "6.0", "6.1", "7.0", "7.1", "8.0", "8.1", "8.2", "8.3", "8.4 (12H141)", "9.0 (13A340)", "9.1 (13B5110e)", "9.2 (13C75)", "9.3 (13E230)", "10.0 (14A345)", "10.1 (14B72)", "10.2 (14C5062c)", "10.3 (14E269)", "11.0 (15A372)", "11.1 (15B87)", "11.2 (15C5092b)", "11.3 (15E5178d)", "11.4 (15F5037c)", "12.0 (16A5288q)", "12.1 (16B5059d)", "12.2 (16E5191d)", "12.3 (16F148)", "12.4", "13.0", "13.1", "13.2", "13.3", "13.4", "13.5", "13.7", "14.0", "14.1", "14.2", "14.4", "14.5", "14.6", "14.7", "14.7.1", "14.8", "15.0", "15.1", "15.2", "15.3.1", "15.3", "15.4", "15.5", "15.6", "15.6.1", "15.7", "16.0", "16.1", "16.2", "16.3", "16.4", "16.4.1", "16.5"}
```
Comment (shamanec) [inline ios/imagemounter/imagedownloader.go:87]: You gotta add it here as well:
```suggestion
	availableVersions = []string{"4.2", "4.3", "5.0", "5.1", "6.0", "6.1", "7.0", "7.1", "8.0", "8.1", "8.2", "8.3", "8.4 (12H141)", "9.0 (13A340)", "9.1 (13B5110e)", "9.2 (13C75)", "9.3 (13E230)", "10.0 (14A345)", "10.1 (14B72)", "10.2 (14C5062c)", "10.3 (14E269)", "11.0 (15A372)", "11.1 (15B87)", "11.2 (15C5092b)", "11.3 (15E5178d)", "11.4 (15F5037c)", "12.0 (16A5288q)", "12.1 (16B5059d)", "12.2 (16E5191d)", "12.3 (16F148)", "12.4", "13.0", "13.1", "13.2", "13.3", "13.4", "13.5", "13.7", "14.0", "14.1", "14.2", "14.4", "14.5", "14.6", "14.7", "14.7.1", "14.8", "15.0", "15.1", "15.2", "15.3.1", "15.3", "15.4", "15.5", "15.6", "15.6.1", "15.7", "16.0", "16.1", "16.2", "16.3", "16.4", "16.4.1", "16.5", "16.6"}
```

**Comment by shamanec** [comment]
> Added new developer images.
> 
> At some point, we should do this automatically.. But unsure the best way to do it with that fuzzy search function.

I think that beginning with iOS 17 DDI will no longer be available in this format and they will be personalized per device so we won't actually have to do this anymore. We just need the amazing @danielpaulus to implement a new way of mounting DDI :P

**Comment by vdelendik** [comment]
I'd recommend to merge 16.6 still very popular nowadays:)

File: ios/imagemounter/imagedownloader.go (hunk 1 above)
Comment (eyJhb) [reply ios/imagemounter/imagedownloader.go:87]: I'll look at this tomorrow, sorry for overlooking it!

**Comment by danielpaulus** [comment]
I am already working on it :-) 
The weather in Germany is getting colder and more rainy, summer is over, expect faster progress now :-D

File: ios/imagemounter/imagedownloader.go (hunk 1 above)
Comment (eyJhb) [reply ios/imagemounter/imagedownloader.go]: Done!

**Comment by eyJhb** [comment]
16.7 should also come at some point. But the upstream repo has not added it yet :)

Looking forward to the 17 progress @danielpaulus ! I can provide cat pictures at any point, if it aids in the development ;)

**Comment by eyJhb** [comment]
Done @shamanec :)

**Pull request merged**